Automattic/vip-go-mu-plugins
# Add global wrapper function for Automattic\VIP\Environment::get_var

## Description

  * Added vip_get_env_var function so it can be available in the runtime.
  * Added unit tests to validate the functionality

## Changelog Description

### vip_get_env_var function

Added function: vip_get_env_var as a shortcut to requiring Automattic\VIP\Environment::get_var

```php
$my_env_var = vip_get_env_var( 'FOO',  'BAR' );
```
If the key, for the custom environment variable ("FOO") does not exist, the default value ( "BAR" ) will be returned.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test


1. Check out PR.
1. Create a user defined environment variable in VIP Cli, or just add a constant definition to wp-config.php
1. put the following code in the index.php file of your active template:
  ```php
    <?php
    // testing vip_get_env_var ( Automattic\VIP\Environment::get_var wrapper )
    $myVar = vip_get_env_var('MY_VAR');
    ?>

    <script>
      console.log('$myVar: <?php echo $myVar; ?> ');
    </script>
  ```
  On a page load, the javascript console should display the value
